python/rikai/spark/sql/codegen/pytorch.py

```diff
@@ -84,6 +85,13 @@ def torch_inference_udf(
                     results.extend(bin_predictions)
                 yield pd.Series(results)
 
+        # Release GPU memory
+        # https://blog.paperspace.com/pytorch-memory-multi-gpu-debugging/?ref=tfrecipes
+        if use_gpu:
```

Comment:
does this need to go in a finally block?

Reply:
Good point. it is always better to do so.